tidb-cloud/changefeed-sink-to-cloud-storage.md

```diff
@@ -18,36 +18,97 @@ This document describes how to create a changefeed to stream data from TiDB Clou
+For **GCS**, before filling **GCS Endpoint**, you need to first grant the GCS bucket access. Take the following steps:
 
-5. Customize **Table Filter** to filter the tables that you want to replicate. For the rule syntax, refer to [table filter rules](https://docs.pingcap.com/tidb/stable/ticdc-filter#changefeed-log-filters).
+1. In the TiDB Cloud console, record the **Service Account ID**, which will be used to grant TiDB Cloud access to your GCS bucket.
+
+    ![gcs_endpoint](/media/tidb-cloud/changefeed/sink-to-cloud-storage-gcs-endpoint.png)
+
+2. In the Google Cloud Platform (GCP) Management Console, create an IAM role for your GCS bucket.
+
+    1. Sign in to the [GCP Management Console](https://console.cloud.google.com/).
+    2. Go to the [Roles](https://console.cloud.google.com/iam-admin/roles) page, and then click **Create role**.
+
+        ![Create a role](/media/tidb-cloud/changefeed/sink-to-cloud-storage-gcs-create-role.png)
+
+    3. Enter a name, description, ID, and role launch stage for the role. The role name cannot be changed after the role is created. 
+    4. Click **Add permissions**. Add the following read-only permissions to the role, and then click **Add**.
+
+        - storage.buckets.get
+        - storage.objects.create
+        - storage.objects.update
+        - storage.objects.get
+        - storage.objects.list
+        - storage.objects.delete
+
+    ![Add permissions](/media/tidb-cloud/changefeed/sink-to-cloud-storage-gcs-assign-permission.png)
+
+3. Go to the [Bucket](https://console.cloud.google.com/storage/browser) page, and choose a GCS bucket you want TiDB Cloud to access. Note that the GCS bucket must be in the same region as your TiDB cluster.
+
+4. On the **Bucket details** page, click the **Permissions** tab, and then click **Grant access**.
+
+    ![Grant Access to the bucket ](/media/tidb-cloud/changefeed/sink-to-cloud-storage-gcs-grant-access-1.png)
+
+5. Fill in the following information to grant access to your bucket, and then click **Save**.
+
+    - In the **New Principals** field, paste the **Service Account ID** of the target TiDB cluster you recorded before.
+    - In the **Select a role** drop-down list, type the name of the IAM role you just created, and then choose the name from the filter result.
+
+    > **Note:**
+    >
+    > To remove the access to TiDB Cloud, simply remove the access that you have granted.
+
+6. On the **Bucket details** page, click the **Objects** tab.
+
+    - To get a bucket's gsutil URI, click the copy button and add `gs://` as a prefix. For example, if the folder name is `test-sink-gcs`, the URI would be `gs://test-sink-gcs/`.
+
+        ![Get bucket URI](/media/tidb-cloud/changefeed/sink-to-cloud-storage-gcs-uri01.png)
+
+    - To get a folder's gsutil URI, open the folder, and then click the copy button add `gs://` as a prefix. For example, if the folder name is `changefeed-xxx`, the URI would be `gs://test-sink-gcs/changefeed-xxx`.
```

Comment:
"if the folder name is `changefeed-xxx`" => if the bucket name is `test-sink-gcs` and the folder name is `changefeed-xxx`